index.d.ts

```diff
@@ -1,4 +1,4 @@
-import * as CommonMediaLibrary from '@svta/common-media-library';
+import { CommonMediaRequest, CommonMediaResponse, RequestInterceptor, ResponseInterceptor } from '@svta/common-media-library/request'
```

Comment:
This change ensures the entire CML package isn't bundled just for the few types that are used.